### DISCUSSION TITLE
Dynamic Content

### DISCUSSION THREAD

**Discussion by ecreatrix** [statamic/cms#5671] [Q&A] [answered]
I'm giving this a test and seem to be missing some basic points. How do I show dynamic content (such as forms or anything inside {{ }} ) on my page? For example, putting {{ now format="Y" }} in my markdown just shows up as {{ now format="Y" }} instead of being parsed. Does it go somewhere else?

**Comment by jackmcdade** [comment, marked as answer]
Answer: https://statamic.dev/antlers#using-antlers-in-content

**Reply by ecreatrix** [reply]
Ah, I knew I was missing something simple. Thanks for clarifying it for me.
